### DISCUSSION TITLE
The shape property of FormBuilderChoiceChip does not take effect

### DISCUSSION THREAD

**Discussion by szsnnet** [danvick/flutter_form_builder#1076] [General]
**The shape property of FormBuilderChoiceChip does not take effect**

FormBuilderChoiceChip(
                          name: 'name',
                          labelPadding: EdgeInsets.symmetric(horizontal: 8.w, vertical: 0.h),
                          selectedColor: Colors.lightBlueAccent.withAlpha(50),
                          selectedShadowColor: Colors.lightBlueAccent.withAlpha(80),
                          backgroundColor: const Color(0xffeeeeee),
                          materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
                          shape: RoundedRectangleBorder(
                            borderRadius: BorderRadius.circular(5),
                          ),
                          spacing: 5.w,
                          visualDensity: VisualDensity.compact,
                          onChanged: (dynamic value) => setState(() {
                                _workerTimeLimitValue = value;
                              }),
                          options: _workerTimeLimit
                              .map(
                                (item) => FormBuilderChipOption(
                                    value: item['value'],
                                    child: Text(
                                      item['title'],
                                      style: TextStyle(
                                          fontWeight: FontWeight.w400,
                                          color: _workerTimeLimitValue == item['value']
                                              ? const Color(0xff3366ff)
                                              : const Color(0xff333333)),
                                    )),
                              )
                              .toList(),
                          decoration: InputDecoration(
                            prefixIcon: Container(
                              width: 78.w,
                              alignment: Alignment.centerLeft,
                              child: const Text(
                               title,
                                style: TextStyle(color: Color(0xff666666)),
                              ),
                            ),
                            suffixIconConstraints: const BoxConstraints(maxHeight: 16),
                         
                            enabledBorder: _border,
                            focusedBorder: _border,
                          )),

**Comment by deandreamatias** [comment]
[This PR](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/pull/1069) solve this, but isn't released to pub.dev yet
